app/src/main/java/org/dhis2/data/forms/dataentry/fields/edittext/CustomTextView.java

```diff
@@ -111,7 +101,6 @@ public CustomTextView(Context context, AttributeSet attrs, int defStyleAttr) {
     public void init(Context context) {
         super.init(context);
         inflater = LayoutInflater.from(context);
-        validators = ((Components) context.getApplicationContext()).appComponent().injectValidators();
```

Comment:
i think you can delete this method inject validators in appcomponent 👍🏻

Reply:
@JaimeToca , he needs it. We can't inject to a view/activity/fragment if it has not its own component and module. And since we are willing to remove custom views, I would avoid adding extra classes to them.